Sources/RInAppMessaging/Router.swift

```diff
@@ -137,4 +230,96 @@ internal class Router: RouterType {
             return rootView
         }
     }
+
+    private func findParentViewForTooltip(_ sourceView: UIView) -> UIView {
+        guard let superview = sourceView.superview else {
+            return sourceView
+        }
+
```

Comment:
I think you can use pattern matching instead of if/else, it would improve the readability:

Reply:
True. Now it looks much better 👍